tutor/commands/config.py

```diff
@@ -255,9 +259,37 @@ def patches_show(context: Context, name: str) -> None:
         print(rendered)
 
 
+@click.command(name="edit", help="Edit config.yml of the current environment")
+@click.pass_obj
+def edit(context: Context, editor: str, save: bool) -> None:
```

Comment:
```suggestion
def edit(context: Context) -> None:
```
Leftover from `-s` and `-e